test/test_retriever.jl

```diff
@@ -79,6 +79,17 @@ function empty_files(path, ext)
     end
 end
 
+function dataset_name_upstream():
```

Comment:
I don't know how this made it's way here in my last commit ` :`

Reply:
Looks like from python :smile: . I will fix it.